include/implGBL/EUTelExcludedPlanes.h

```diff
@@ -0,0 +1,28 @@
+#ifndef EUTELEXCLUDEPLANES_H
```

Comment:
Why is EUTelExcludedPlanes suddenly in implGBL? This is supposed to be a EUTel generic class! It must only deal with the mechanism of excluding planes for fitter or similar processors...